support-workers/src/main/scala/com/gu/support/workers/lambdas/CreateZuoraSubscription.scala

```diff
@@ -261,12 +271,14 @@ object CreateZuoraSubscription {
 
 object DigitalSubscriptionGiftRedemption {
 
-  def maybeDigitalSubscriptionGiftRedemption(product: ProductType, paymentMethod: Either[PaymentMethod, RedemptionData]) = {
+  def maybeDigitalSubscriptionGiftRedemption(
+    product: ProductType,
+    paymentMethod: Either[PaymentMethod, RedemptionData]
+  ): Option[RedemptionData] =
```

Comment:
scalastyle was complaining it's a public method with no return type, and after adding it was >160 width